backend/DevUp/DevUp.Domain/Entities/IEntityId.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace DevUp.Domain.Entities
+{
+    public interface IEntityId<TId>
+    {
```

Comment:
This should probably be an `abstract class` as we can sneak some shared behavior right in. For example, all `EntityId`'s needs to implement `IEquatable<TId>`, otherwise we won't be able to compare them easily.